interpreter/src/main/scala/pl/touk/nussknacker/engine/definition/parameter/StandardParameterEnrichment.scala

```diff
@@ -21,13 +21,22 @@ object StandardParameterEnrichment {
   private def enrichParameter(original: Parameter, parameterConfig: ParameterConfig): Parameter = {
     val parameterData = ParameterData(original.typ, Nil)
     val finalEditor = original.editor.orElse(EditorExtractor.extract(parameterData, parameterConfig))
-    val finalValidators = (original.validators ++ extractAdditionalValidator(parameterData, parameterConfig, finalEditor)).distinct
+    val finalValidators = determineValidatorss(original, parameterConfig, parameterData, finalEditor)
     val isOptional = original.isOptional
     val finalDefaultValue = original.defaultValue.orElse(DefaultValueDeterminerChain.determineParameterDefaultValue(
       DefaultValueDeterminerParameters(parameterData, isOptional, parameterConfig, finalEditor)))
     original.copy(editor = finalEditor, validators = finalValidators, defaultValue = finalDefaultValue)
   }
 
+  private def determineValidatorss(original: Parameter, parameterConfig: ParameterConfig, parameterData: ParameterData, finalEditor: Option[ParameterEditor]) = {
+    // In case if someone forget to specify validators and used only Parameter() or Parameter.optional(), we add missing forgotten validators
+    // It can be tricky, because we have no option to specify that we don't want those "standard" validators
```

Comment:
So, if someone specifies own validators we skip 'standard' ones? Maybe it would be better to add explicitly some flag, like 'skipStandardValidators'? If you need this change now for some reasons - let's add it to changelog/migration guide, as this can be tricky change in behaviour?

Reply:
We currently have such a case: in `GenericTypedJsonSourceFactory` we have parameter `type` with `JsonParameterEditor` which causes adding `JsonValidator` but it is overriden by explicit list of validators. I personally don't like adding those "standard" validators at all and don't want to adding extra fields to `Parameter` for that. WTYT about removing this implication: `JsonParameterEditor` ->  `JsonValidator` and let's developer decide if he/she want to add only editor or both editor and validator?

Reply:
ok, I looked a bit at it, and I'm in favour of not adding these validators at all ;)

Reply:
Changed but for now I only removed `JsonValidator` from those `EditorBasedValidatorExtractor` - must be turned on explicitly because I see a pain for developers to defined both editor and validator for "fixed-values" case. At the end we should rather add something like `ParameterEditor.validators` method and in `Paramter` collect both `validators` and `editor.validators` but don't want to change it in this PR